### ISSUE TITLE
Missing documentation about the mode line. What is the "AP:00%"? Also, space needed.

### ISSUE THREAD

**Issue by echosa** [xcwen/ac-php#87]
I don't see any documentation about what ac-php adds to the mode line. What is the "AP:00%"? 

Also, there needs to be a space at the beginning. It should be " AP:00%". Otherwise, there's no space between it and the previous mode. In my case, I see "companyAP:00%" instead of "company AP:00%".

**Comment by xcwen** [comment]
00%  : the process of gen tags .

**Comment by echosa** [comment]
I see. I don't think it's working properly. I see three issues:

1.) I was only seeing that in the mode line for one PHP buffer, not the others. Even then, when I ran `ac-php-rename-tags-all`, the percentage never changed.

2.) You should still add a space before the "AP:" part.

3.) The README and documentation should be updated to explain the mode line.

**Comment by xcwen** [comment]
run ac-php-rename-tags-all, the percentage never changed.?!

**Comment by echosa** [comment]
Ok, I just restarted Emacs to start clean and tried again. The percentage does work. Also, the percentage didn't show at all until I ran the remake, which makes sense. The remake ran, and I did watch the percentage go up. So it does seem to be working. That's pretty cool to see the remake status like that.

However, it ran twice. It went 0 to 99, then 0 to 99 again. Not sure why. 

So the only issues are not having a space before it and it not being documented (so others don't have the same question as me here). :-)
